src/state/slices/validatorDataSlice/const.ts

```diff
@@ -1 +1,4 @@
-export const SHAPESHIFT_VALIDATOR_ADDRESS = 'cosmosvaloper199mlc7fr6ll5t54w7tts7f4s0cvnqgc59nmuxf'
+export const SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS =
```

Comment:
We use `constants.ts` as the filename for storing constants everywhere else - can we please update this to keep it consistent?